setup.bat

```diff
@@ -1,6 +1,6 @@
 @echo off
 pip install virtualenv
-virtualenv .venv
-call .venv/Scripts/activate.bat
+virtualenv venv
+call venv/Scripts/activate.bat

```

Comment:
I saw it was a best practice to name the directory `.venv`

Note: if you change it here, change it also in run.bat